### PR TITLE
Move to automatic generation of UIColors from Elan

### PR DESCRIPTION
**Pull request by orta**
This clones Elan locally, then parses it's JSON files and generates our own files based on that. I'll highlight in the diff where this is happening. 

I've removed the `pods` dir. As I've removed the only dependency (EDColors) we really don't need to have all that cruft in the repo.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,6 +1,92 @@
 desc "Runs the specs [EMPTY]"
-task :spec do
```
Comment (orta) [inline Rakefile]: this is where the interesting work is

**Comment by orta** [comment]
Giving up on @alloy reviewing ;)

File: UnitTests/DebugUnitTests.m [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    sut = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 44, 44)];
+});
+
+describe(@"debug", ^{
+    it(@"blue is aesthetically pleasing", ^{
+        sut.backgroundColor = [UIColor debugColourBlue];
+        expect(sut).to.haveValidSnapshot();
+    });
+
+    it(@"puple is exceedingly purple", ^{
```
Comment (sarahscott) [inline UnitTests/DebugUnitTests.m]: purple*

**Comment by sarahscott** [comment]
:ok_hand: looks good to me! just that one typo

File: UnitTests/DebugUnitTests.m (hunk 2 above)
Comment (orta) [inline UnitTests/DebugUnitTests.m]: haha, yep, nice spot

**Pull request merged**